src/openpgp.js

```diff
@@ -93,6 +94,41 @@ export function destroyWorker() {
   asyncProxy = undefined;
 }
 
+//////////////////////////
+//                      //
+//   indutny/elliptic   //
+//                      //
+//////////////////////////
+
+
+let elliptic;  // instance of the indutny/elliptic
+
+/**
+ * Load elliptic by path or from node_modules
+ * @param {String} path relative path to elliptic browserified package
+ */
+export async function loadElliptic(path) {
```

Comment:
I would put this function somewhere else, maybe indutnyKey.js?

Reply:
I could move it here, but should we move `loadScript` from openpgp? We could use it to load other rarely used code, like md5 hash.

Reply:
moved to lightweight_helper.js